### ISSUE TITLE
Invalid CSS property value in Cart in Storefront theme

### ISSUE THREAD

**Issue by Jpegtobbe** [woocommerce/storefront#601]
There is invalid css in the Storefront theme, on the cart page. On the remove icon.
In the class table.cart td.product-remove a.remove ->  top: -ms(-2);
Got to be a typo? -ms(-2) is not valid css. I guess thats something from the "modularscale"-lib?

Go to the cart page. View the page in mobile mode. Then inspect the a.remove element.

WC Version: 3.0.8
WP Version: 4.8
Storefront Version: 2.2.3

**Comment by jameskoster** [comment]
> I guess thats something from the "modularscale"-lib?

Correct. I think a space is required between `-` and the `ms()` function for it to compile properly.
